### ISSUE TITLE
I think it's not working, I tried with latest Sketch version

### ISSUE THREAD

**Issue by jitendravyas** [mfouquet/Color-Copier#3]

**Comment by listao** [comment]
Yeah, this plugin does not work for me either. It did before, but doesn't seem to be on the latest version of Sketch. Oddly, it's not even considered a "depreciated" plugin.

**Comment by tobystokes** [comment]
It's only the Hex colour mode that doesn't work for me, others like RGB seem fine
